src/utils/ad-utils.ts

```diff
@@ -185,15 +185,15 @@ export const restrictDecimalPlace = (
  * @param {string[]} errorCodes - The array of error codes.
  * @returns {string} - The error message.
  */
-export const getEligibilityErrorMessage = (errorCodes: string[]) => {
+export const getEligibilityErrorMessage = (errorCodes: string[], localize: TLocalize) => {
     const errorMessages: { [key: string]: string } = {
-        completion_rate: 'Your completion rate is too low for this ad.',
-        join_date: "You've not used Deriv P2P long enough for this ad.",
+        completion_rate: localize('Your completion rate is too low for this ad.'),
+        join_date: localize("You've not used Deriv P2P long enough for this ad."),
     };
 
     if (errorCodes.length === 1 && errorMessages[errorCodes[0]]) {
         return errorMessages[errorCodes[0]];
     }
 
-    return "The advertiser has set conditions for this ad that you don't meet.";
+    return localize("The advertiser has set conditions for this ad that you don't meet.");
```

Comment:
We need to get rid of the localize here where we wrapped this function 😅 
<img width="667" alt="Screenshot 2024-08-06 at 10 55 02 AM" src="https://github.com/user-attachments/assets/af653877-dc7c-44aa-9213-316c03edd3be">

Reply:
It's already removed @ameerul-deriv.